### ISSUE TITLE
how to test my pictures?

### ISSUE THREAD

**Issue by mumuyanyan** [csmliu/STGAN#29]
hi, thanks for your great project.
I have a question about the inference of my images. 
how to change the attributes of my images by using the model?

**Comment by csmliu** [comment]
Hi, the model is trained on aligned CelebA dataset, thus you should align and crop the image first, and then use the model to predict the desired image.

By the way, I wrote a small demo, which can capture video image from the USB Camera, align the image (using dlib 5-point detector), generate output image and put it back to the input frame.
The demo file is uploaded here for reference, and a detailed illustration will be added to the repo in several days.

Just put the uncompressed files into the root path of STGAN, and try the demo with Python 3.x and Tensorflow<=1.12.
[STGAN_demo.zip](https://github.com/csmliu/STGAN/files/3993975/STGAN_demo.zip)

**Comment by Blue-Clean** [comment]
@csmliu  thanks for your nice work. I wanna know how to evaluate your model. For example, the psnr and ssim of reconstructions images is get from all the text images? From 182638 to 202599one?

**Comment by csmliu** [comment]
Yes, the PNSR and SSIM indices are obtained from the official test set (from 182638 to 202599) by setting the difference attribute vector as 0.
If you've generated all the test images (under the instruction given in https://github.com/csmliu/STGAN/tree/master/att_classification), you can put the attached file in `output/experiment_name` folder and just run it.
[psnr_ssim.zip](https://github.com/csmliu/STGAN/files/4000529/psnr_ssim.zip)
